docs/admin/external-auth.md

```diff
@@ -191,21 +191,21 @@ CODER_EXTERNAL_AUTH_0_ID=primary-github
 CODER_EXTERNAL_AUTH_0_TYPE=github
 CODER_EXTERNAL_AUTH_0_CLIENT_ID=xxxxxx
 CODER_EXTERNAL_AUTH_0_CLIENT_SECRET=xxxxxxx
-CODER_EXTERNAL_AUTH_0_REGEX=github.com/org
+CODER_EXTERNAL_AUTH_0_REGEX=github\.com/org
```

Comment:
Should we add a comment that it is only required if using multiple external auth for Github belonging to different organizations?

Reply:
I don't think so, we want the regex to be correct, and dropping the \ only opens the user up to the same issue if they do one day decide to do multiple external auth.

Reply:
I mean the `CODER_EXTERNAL_AUTH_0_REGEX` field it self. It is optional and only comes into action when someone have multiple GitHub auths.